src/boost_histogram/_internal/axis.py

```diff
@@ -155,30 +157,47 @@ def _repr_kwargs(self):
         """
 
         ret = ""
-        if self.options.growth:
+        if self.traits.growth:
             ret += ", growth=True"
-        elif self.options.circular:
+        elif self.traits.circular:
             ret += ", circular=True"
         else:
-            if not self.options.underflow:
+            if not self.traits.underflow:
                 ret += ", underflow=False"
-            if not self.options.overflow:
+            if not self.traits.overflow:
                 ret += ", overflow=False"
 
         return ret
 
     @property
     def options(self):
         """
+        DEPRECATED: use .trails instead.
         Return the options.  Fields:
           .underflow - True if axes captures values that are too small
           .overflow  - True if axes captures values that are too large
                        (or non-valid for category axes)
           .growth    - True if axis can grow
           .circular  - True if axis wraps around
         """
+        warnings.warn("DEPRECATED: use .traits instead", FutureWarning)
         return self._ax.options
 
+    @property
+    def traits(self):
+        """
+        Get traits for the axis - read only properties of a specific axis.
+        """
+        return Traits(
+            self._ax.traits_underflow,
+            self._ax.traits_overflow,
+            self._ax.traits_circular,
+            self._ax.traits_growth,
+            self._ax.traits_continuous,
+            self._ax.traits_inclusive,
```

Comment:
Perhaps leave that out for now? What use would it have?

Reply:
Yes, probably best. I was just adding things I saw in the interface to Boost.Histogram that looked remotely useful, since `continuous` was useful and missing.